asterius/src/Asterius/Builtins/Env.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Asterius.Builtins.Env
+  ( envImports,
+    envCBits,
+  )
+where
+
+import Asterius.EDSL
+import Asterius.Types
+import qualified Asterius.Types.SymbolMap as SM
+
+envImports :: [FunctionImport]
+envImports =
+  [ FunctionImport
+      { internalName = "__asterius_getProgArgv",
+        externalModuleName = "posix",
+        externalBaseName = "getProgArgv",
+        functionType =
+          FunctionType
+            { paramTypes = [F64, F64],
+              returnTypes = []
+            }
+      }
+  ]
+
+envCBits :: AsteriusModule
+envCBits = envArgvBuf <> envGetProgArgv
+
+envArgvBuf :: AsteriusModule
```

Comment:
```suggestion
-- | Data segment used to store runtime arguments. In normal native programs,
the arguments passed to a program @argv@ exist in special memory pages and are
managed by the OS kernel. Since we cannot do this here, we instead reserve a
data segment of 1KB of memory to be used for this purpose (see @getProgArgv@ in
@rts/node/default.mjs@).
envArgvBuf :: AsteriusModule
```

Reply:
Missing `--` prefix for rest of the lines.

Reply:
:facepalm: so true. I just updated the suggestion.